nameservice/tutorial/00-intro.md

```diff
@@ -63,9 +62,10 @@ Through the course of this tutorial you will create the following files that mak
 Start by creating a new git repository:
 
 ```bash
-mkdir -p $GOPATH/src/github.com/{ .Username }/nameservice
-cd $GOPATH/src/github.com/{ .Username }/nameservice
+mkdir -p nameservice
```

Comment:
```suggestion
mkdir nameservice
```

would suggest removing p, without multiple ply in the tree it is only preventing errors when the dir already exists (an error you would presumably want here)